internal/storage/caching/mock_gcscaching/mock_stat_cache.go

```diff
@@ -61,23 +61,6 @@ func (m *mockStatCache) AddNegativeEntry(p0 string, p1 time.Time) {
 	}
 }
 
-func (m *mockStatCache) CheckInvariants() {
```

Comment:
This has been removed because statCache does not have a CheckInvariants() function anymore. There is no alternatives to this function.